.github/workflows/downstream.yml

```diff
@@ -0,0 +1,33 @@
+name: Downstream
+on:
+  push:
+    branches:
+      - main
+    tags: '*'
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: downstream ClimaCoupler.jl
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v1
```

Comment:
```suggestion
      - uses: julia-actions/setup-julia@v2
```

Reply:
does `julia-actions/julia-buildpkg@v1` need to be updated too?

also, I copied this directly from ClimaUtilities, which is still using v1. Maybe we should update there too

Reply:
Yes, I already opened a PR there.

And no, `julia-buildpkg` is still at version 1.